doc/references.bib

```diff
@@ -2065,6 +2065,18 @@ @inproceedings{Fadnavis2020
   url       = {https://proceedings.neurips.cc/paper_files/paper/2020/file/bc047286b224b7bfa73d4cb02de1238d-Paper.pdf},
 }
 
+@inproceedings{Fadnavis2024,
```

Comment:
Please, read the documentation about where to place references: newer references go closer to top:
https://github.com/dipy/dipy/blob/master/doc/devel/bibliography.rst

Reply:
Thank you for pointing it out @jhlegarreta. Will fix it right away.